src/jit/gentree.h

```diff
@@ -349,6 +349,8 @@ struct GenTree
 
 #endif // FEATURE_ANYCSE
 
+    unsigned char     gtLIRFlags; // Used for nodes that are in LIR.
+
 #if ASSERTION_PROP
```

Comment:
We should (eventually?) have a better comment here, stating what all the acceptable flags are (maybe giving a type, or a set of #defines?)